node/core/approval-voting/src/lib.rs

```diff
@@ -2463,6 +2532,10 @@ async fn issue_approval<Context>(
 	candidate_hash: CandidateHash,
 	ApprovalVoteRequest { validator_index, block_hash }: ApprovalVoteRequest,
 ) -> SubsystemResult<Vec<Action>> {
+	let span = state.spans.get(&block_hash).unwrap();
+	let mut issue_approval_span =
+		span.child("issue-approval").with_stage(jaeger::Stage::ApprovalChecking);
+	issue_approval_span.add_string_tag("candidate-hash", format!("{:?}", candidate_hash));
```

Comment:
why not use `.with_candidate(candidate_hash)` ?

Reply:
same as above, yep, will do 👍🏼 ⚡